src/IonParserBinaryRaw.ts

```diff
@@ -155,13 +155,13 @@ export class ParserBinaryRaw {
         }
     }
 
-    private readVarUnsignedInt() : number {
+    static readVarUnsignedFrom(input: BinarySpan) : number {
```

Comment:
The `readUnsignedInt` and `readVarUnsignedLongInt` methods both rely on internal state from the `ParserBinaryRaw` class. In order to make them testable, I've moved their core logic out into two new static methods: `readUnsignedIntFrom` and `readVarUnsignedLongIntFrom`. These methods allow you to bring your own state, making them easier to test.